rosdep/ruby.yaml

```
@@ -111,3 +120,8 @@ ruby-sass:
 ruby1.9.3:
   ubuntu:
     precise: [ruby1.9.3]
+utilrb:
+  debian:
+    gem: metaruby
+  ubuntu:
+    gem: metaruby
```

Did you mean `utilrb`?
```suggestion
    gem: utilrb
  ubuntu:
    gem: utilrb
```

---

You are right. Classical copy&paste error. Fixed in https://github.com/ros/rosdistro/pull/28155/commits/9ad99c672b5ede65ea5ae7c8de3c6d6cf80c507e.